hack/test-go.sh

```diff
@@ -303,6 +303,14 @@ reportCoverageToCoveralls() {
   fi
 }
 
+checkFDs() {
+  # several unittests panic when httptest cannot open more sockets
+  # due to the low default files limit on OS X.  Warn about low limit.
+  fileslimit = $(ulimit -n)
+  if [[ fileslimit -lt 1000 ]]; then
+echo "WARNING: ulimit -n (files) should be at least 1000, is $fileslimit, may cause test failure"; fi
```

Comment:
Indentation is strange here. Should be
```bash
  if [[ fileslimit -lt 1000 ]]; then
     echo "WARNING: ulimit -n (files) should be at least 1000, is $fileslimit, may cause test failure"
  fi
```
I'm also not a huge fan of the warning message.